docs/pages/inboxes/disappearing-messages.md

```diff
@@ -0,0 +1,97 @@
+5. If the message hasn't expired, the app displays the message in the messaging UI.
+6. If the message has expired, the message is removed from (or not displayed in the first place in) the conversation participants’ app UIs and is deleted from the apps’ local storage.
+
+### Enable disappearing messages for a conversation
+
+When creating or updating a conversation, enable users with appropriate permissions to set disappearing message expiration conditions.
+
+This includes setting the following conditions expressed in nanoseconds (ns):
+
+- `from_ns`: Starting timestamp from which the message lifespan is calculated
+- `in_ns`: Duration of time during which the message should remain visible to conversation participants
+
+For example:
+
+1. Set `from_ns` to the current time, such as `1738620126404999936` (nanoseconds since the Unix epoch of January 1, 1970).
+2. Set `in_ns` to the message lifespan, such as 1800000000000000 (30 minutes).
+3. Use `from_ns` and `in_ns` to calculate the message expiration time of `1738620126404999936 + 1800000000000000 = 1740420126404999936`.
+
+To learn more see [conversation.rs](https://github.com/xmtp/libxmtp/blob/main/bindings_node/src/conversation.rs#L49).
+
+### Send a disappearing message
+
+When sending a message, it abides by message expiration conditions set for the conversation. For example:
+
+```text
+// Example: disappearing message with an expiration time based on a 
+// countdown start time and expiration countdown
+```
+
+### Receive a disappearing message
+
+On the receiving side, your app needs to:
+
+1. Check for disappearing message metadata set for the conversation. For example:
+    
+    ```text
+    // Example: check for disappearing message on the message's parent conversation
+    ```
+    
+2. Display the message if the current time is before the expiration time. For example:
+    
+    ```text
+    // Example: Check is the current time is before the expiration time
+    ```
+    
+    Here are some tips for displaying disappearing messages in your app to ensure that users understand which messages use the feature and what to expect.
+    
+    - **Use a distinct visual style**: You might style disappearing messages differently from regular messages (e.g., a different background color or icon) so users understand they're short-lived.
+    - **Display a countdown**: Show a small countdown that indicates how much time remains before the message disappears.
+
+### Periodically check message expiration time
+
+A worker runs every two seconds to clean up expired disappearing messages. The worker also updates the message expiration policy to align with any updated policy values. For example:
+
+```text
+// Example: Use the worker that runs every two seconds 
+// to clean up expired messages based on group disappearing settings. 
+// The worker will also update the message expiration policy to align 
```

Comment:
```suggestion
```
Tbh the worker does not do this, the worker only checks the messages if needed to be deleted, the settings will be received via a commit that includes the conversation's update info.